lib/rspec/core/output_wrapper.rb

```diff
@@ -3,17 +3,25 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        output.respond_to?(name, priv)
+      end
+
+      def method_missing(name, *args, &block)
+        output.method_missing(name, *args, &block)
+      end
```

Comment:
Breaks the deprecation stream support for string filenames (which was publicly documented beforehand), and the support for adding formatters with string filenames which was implicitly allowed beforehand (some even had the option of multiple string to file names, which is what the map is for in formatter loadeD)